src/utils/columnUtils.ts

```diff
@@ -164,7 +164,7 @@ export function canEdit<R, SR>(column: Column<R, SR>, row: R): boolean {
   if (typeof column.editable === 'function') {
```

Comment:
I wonder if we should move `canEdit` to `selectedCellUtils.ts`. It is only used once

Reply:
Done.